.github/workflows/toolbox-deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: "Toolbox Deploy"
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    if: startsWith(github.event.release.name, '@neo4j/graphql-toolbox')
+
+    runs-on: ubuntu-latest
+
+    environment: aws
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: refs/tags/${{ github.event.release.tag }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+      - name: Install dependencies
+        run: yarn
+      - name: Build graphql-toolbox
+        run: yarn build
```

Comment:
Not entirely sure, but it could possibly be better to add the `--production` flag: `yarn build --production`

Reply:
This is already implied, as `build` is a script in `package.json`: `cross-env NODE_ENV=production webpack --config webpack.config.js`